Sources/BluemixAppID/UserAttributeManager.swift

```diff
@@ -106,8 +109,8 @@ public class UserAttributeManager {
             }
         })
 
-        if attributeValue != nil {
-            request.write(from: attributeValue!)// add attributeValue to body if setAttribute() was called
+        if let attValue = attributeValue {
```

Comment:
I think convention is to name the unwrapped variable the same, so `if let attributeValue = attributeValue {}`